program-test/src/lib.rs

```diff
@@ -7,20 +7,6 @@ use {
     log::*,
     solana_banks_client::start_client,
     solana_banks_server::banks_server::start_local_server,
-    solana_program::{
```

Comment:
FTR: The reason I originally explicitly used `solana_program` here instead of the `solana_sdk` re-imports is that I was envisioning a future release where we rip out the `solana_sdk` re-imports entirely.  Maybe that's a pipe dream though

Reply:
Yeah, I figured it was something like that, it was a nit pain point when I would add new exports and then find they were already added in the other.  Also, because we aren't at the point where we can use `solana_program` it was confusing why they were split and if there was some larger reason to include both.  We can easily switch to `solana_program` later if/when we are able to.